src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -89,7 +89,7 @@ export function getEnrollmentDetails(
     blocks.push(
       <>
         <strong>You applied on: </strong>
-        {moment(applicationDate).format('MMMM D, YYYY')}
+        {format(Date.parse(applicationDate), 'MMMM d, yyyy')}
```

Comment:
`applicationDate` is the first argument in `getEnrollmentDetails`. Even though there isn't a unit test specifically for this function, it is used in `getWarningStatus()`, and the second argument is passed directly through.

https://github.com/department-of-veterans-affairs/vets-website/blob/8af6da8f01587d87d66e12ddde79b0e4f22e81b7/src/applications/hca/tests/enrollment-status-helpers.unit.spec.js#L40-L45